### PR TITLE
chore: initialize tailwind

### PR DESCRIPTION
**Pull request by ishoborabyose**
###  PR DESCRIPTION

- This PR initializes TailwindCSS in the project

### TASKS COMPLETED

- [x] Installing TailwindCSS
- [x] Insert in config
- [x] Install the automatic Class Sorting with Prettier
- [x] Initialize the font family

### HOW CAN THIS PR BE TESTED

```
git clone https://github.com/TheGymRwanda/unit8-maroon
git checkout ch/tailwind-installation
npm install
npm run dev

```

### BEFORE SUBMITTING YOUR PR, ENSURE IT HAS THE FOLLOWING

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard to understand areas
- [x] My changes generate no new warnings

### TICKET NUMBER

#OO4: https://www.notion.so/apeunit/Initialize-TailwindCSS-11dc247d57304278bacc8a8ee35bbe9d

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -1,26 +1,3 @@
...
-    color-scheme: dark;
-  }
-  body {
-    color: white;
-    background: black;
-  }
-}
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (Delyc) [inline styles/globals.css:3]: Not seeing where you configured font family in global css. Work on it!

**Review by Delyc — CHANGES_REQUESTED** [review]

File: styles/globals.css (hunk 1 above)
Comment (ishoborabyose) [reply styles/globals.css:3]: Thank you for your review, let's do it ASAP

File: tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        "18" : "4.5rem",             //72px
+        "10.5" :"2.625rem",                  //42px
+        "4.5" : "1.125rem"             //18px
+      },
+      fontFamily: {
+        'Neufile': ['Neufile', 'sans-serif'],        
+      },
+      spacing:{
+        "24.5" : "6.1513rem",
+      },
```
Comment (haroldmud) [inline tailwind.config.js:36]: Tenderly add comments that show the `rem` value in `px`

**Review by haroldmud — APPROVED** [review]
Nice job, Your work on this Task is well done, though there is still some change you need to work on for it to be perfect, check out the comment below.

**Comment by Delyc** [comment]
@ishoborabyose I wonder why your PR is not checked on ```My changes generate no new warnings```, if it has warnings fix them or check it

File: styles/globals.css (hunk 1 above)
Comment (Delyc) [reply styles/globals.css:3]: Looks good now!

**Review by Nkaka23dev — APPROVED** [review]
Great! Your job is complete.

**Review by Delyc — APPROVED** [review]

**Review by haroldmud — APPROVED** [review]
## APPROVED 🎉 

Your work is now complete, nothing else to say other than it's time to merge it 💪

**Pull request merged**